### ISSUE TITLE
Modals with component: cannot access content variables

### ISSUE THREAD

**Issue by Gyoo** [valor-software/ngx-bootstrap#2448]
Greetings,

I am trying to use the modal component using [this part of the demo](http://valor-software.com/ngx-bootstrap/#/modals#service-component), however I cannot access the fields in my component when constructing the modal.

Here's the code :

**Component calling the modal**
```
cancel() {
    this.bsModalRef = this.modalService.show(ConfirmComponent);
    this.bsModalRef.content.title = 'Test'; // <== Property "title" does not exist on type 'Object'.
  }
```

**ConfirmComponent**
```
import {Component, EventEmitter, OnInit, Output} from '@angular/core';
import {BsModalRef} from 'ngx-bootstrap/modal';

@Component({
  selector: 'app-confirm',
  templateUrl: './confirm.component.html',
  styleUrls: ['./confirm.component.sass']
})
export class ConfirmComponent {

  public title: string;
  public message: string;

  @Output() confirm: EventEmitter<any> = new EventEmitter();

  constructor(public bsModalRef: BsModalRef) {}

}
```

Because of this error in the cancel() method, the code fails to compile and I can't access the app at all.

I'm using Angular 4.3.1

**Comment by IlyaSurmay** [comment]
which version of ngx-bootstrap are you using? that issue was fixed in 1.8.1

**Comment by Gyoo** [comment]
Ah, looks like I was still using the 1.8.0 version. I upgraded to 1.8.1 and I can now access my fields correctly. Sorry for the inconvenience.
